bitnami/external-dns/Chart.yaml

```diff
@@ -24,4 +24,4 @@ sources:
   - https://github.com/kubernetes-sigs/external-dns
   - https://github.com/bitnami/containers/tree/main/bitnami/external-dns
   - https://github.com/kubernetes-sigs/external-dns
-version: 6.11.3
+version: 6.11.4
```

Comment:
Since this is adding a new feature, we should increase the minor instead (e.g. to `6.12.0`). Could you change that?